src/xdocs/google_style.xml

```diff
@@ -2351,6 +2345,16 @@
           <i>name_of_the_check</i> is the actual name of check to be suppressed in lowercase
           format.
         </p>
+        <p>
+          To suppress a check using
+          <a href="filters/suppresswithplaintextcommentfilter.html#SuppressWithPlainTextCommentFilter">
+            SuppressWithPlainTextCommentFilter
+          </a>
+          add the offComment format you specified in the <code>offCommentFormat</code>
+          property before the beginning of the code block and the onComment
+          format you specified in the <code>onCommentFormat</code>
+          property after the end of the code block.
+        </p>
```

Comment:
I don't think we need this update at all, since JSNI is explicitly mentioned in the style guide.

Reply:
done